assembl/static2/js/app/components/debate/common/announcement.jsx

```diff
@@ -23,8 +23,29 @@ type SentimentsCounts = {
   }
 };
 
+type Post = {
+  node: {
+    sentimentCounts: {
+      [string]: number
+    },
+    publicationState: string
+  }
+};
+
 type Posts = {
-  edges: Array<{ node: { sentimentCounts: { [string]: number }, publicationState: string } }>
+  edges: Array<Post>
+};
+
+type AnnouncementContent = {
+  __typename: string,
+  body: string,
+  title: string
```

Comment:
only body is required here!

Reply:
Ok I figured we have a ```noTitle``` boolean always set to true...